src/access/wac.ts

```diff
@@ -92,6 +98,25 @@ async function getActorClassAccess(
   return aclAccessToUniversal(wacAccess);
 }
 
+async function getActorAccessAll(
+  resource: WithServerResourceInfo,
+  accessEvaluationCallback: typeof getAgentAccessAllWac,
+  options: Partial<typeof internal_defaultFetchOptions>
+): Promise<AgentWacAccess | null> {
+  const resourceAcl = await internal_fetchAcl(resource, options);
+  const wacAgentAccess = accessEvaluationCallback(
+    Object.assign(resource, { internal_acl: resourceAcl })
+  );
+  if (wacAgentAccess === null) {
+    return null;
+  }
+  const result: AgentWacAccess = {};
+  for (const [webId, wacAccess] of Object.entries(wacAgentAccess)) {
```

Comment:
`Object.entries` [is not supported by IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#browser_compatibility), so maybe you could mention this in the README among the list of polyfills as well?

Reply:
4245918a0f69233fb750a25120c2a5a8317d2a3f